phases/5/README.md

```diff
@@ -1,3 +1,33 @@
 # Phase 5
 
-The career prep phase supports learners with interview readiness and overarching career development. Learners in Phase 5 participate in mock technical interviews and career development workshops, while being supported in resume and online presence development, networking skills, and overall accountability support during the job search process.
+Career Prep supports learners with interview readiness, job placement support and overarching career development. Learners are responsible for working to become fully employed by the end of their time at the Guild. Since each learner’s application and job search strategy may vary, learners are responsible for developing weekly work plans that reflect their job hunt strategy and submitting it as gist.
+
+## High Level Skills
+In this phase you will: 
+- Practice realtime problem solving in mock technical interviews
+- Attend career development workshops 
+- Create and refine all job search materials 
+- Develop networking and negotiation skills
+
+## Accountabilities
+Weekly work plans should align with one of the four main accountabilities below. Examples of objectives that would fulfill each accountability are also listed below:
+- Actively apply for jobs 
+  - Submit at least 10-15 applications per week
+  - Attend 1-2 networking events per week (includes meetups)
+  - Attend scheduled 1:1’s with Outbound Recruiter
+- Prepare for Technical Interviews
+  - Complete at least 1 mock interview per week
+  - Complete at least 1 Hacker Rank challenge per week
+- Participate in Phase 5 Career Prep supports
+    - Attend scheduled workshops
+- Continue to develop technical skills
+    - Work on portfolio site
+    - Take on contract work through Upwork, fiverr, or other freelance website
+
+## Weekly Work Plan Specifications
+You are accountable for setting your own objectives for the week on Monday, sharing them via gist in your phase channel, crushing them, and circling back to assess on how the week went during Friday reflection.
+
+Work plans must include specific objectives based on the accountabilities above and follow the format below: 
+- [ ] Artifact is a gist of your weekly workplan
```

Comment:
for the time being I am trying to avoid rendering checkboxes with `- [ ] lorem ipsum` because it allows you to check them but they're not persisted like the skills checkboxes so it's inconsistent and confusing. i ripped them all out a while back. Can you remove all the `[ ] `?